docs/sdk/client-side-sdks/javascript.md

```diff
@@ -185,6 +187,32 @@ dvcClient.flushEvents(() => {
 })
 ```
 
+## Subscribing to SDK Events
+
+The SDK can emit certain events when specific actions occur which can be listened on by subscribing to them:
+
+```javascript
+dvcClient.subscribe('variableUpdated:*', (key: string, variable: DVCVariable) => {
+    // key is the variable that has been updated
+    // variable is the DVCVariable object containing the new value
+    if (variable.value === 'new_value') {
```

Comment:
not sure what this check is doing.

this method is only ever called if the variable value has changed

Reply:
maybe just do 
`console.log(`New variable value for variable ${key}: ${variable.value}`)` or something?

It just seems confusing that the value is being checked here and then the comment afterwards implies that because of this check we now know that the variable value has changed